### ISSUE TITLE
Carla Chronicles quiz solution hard to follow, use mathquill to improve

### ISSUE THREAD

**Issue by jchernus** [udacity/isdc_issues_tracker#199]
https://classroom.udacity.com/nanodegrees/nd113/parts/ad7e52df-3387-4252-b4bd-9c40aecb8093/modules/14fc0233-3169-4ca5-8255-ed921d2d973a/lessons/56980e75-08e4-4b3c-ba7f-58f10e963980/concepts/81f5107e-bacb-4430-9f06-1bc179bbe82a

The sharpest turn that Carla can navigate successfully has a radius of 75m. 12 = 30^2/r 12 = 900/r 12r = 900 r = 900/12 r = 75m

Is a little bit hard to follow. Mathquill can help format the equations to be more readable. Also, use capital R since that's what you had in the question.

**Comment by JonathanKSullivan** [comment]
Thank you, I have made the suggested updates.
